lib/facterdb.rb

```diff
@@ -11,6 +11,13 @@ def self.database
     @database ||= "[#{facterdb_fact_files.map { |f| read_json_file(f) }.join(',')}]\n"
```

Comment:
By the way this should have been probably in a synchronized block, but that's another story.